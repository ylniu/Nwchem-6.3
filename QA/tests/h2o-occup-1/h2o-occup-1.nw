
echo

start h2o-occup-1

memory 1000 mb

geometry units au noautosym noautoz
 O  0.00000000     0.00000000     0.23766060
 H -1.43910775     0.00000000    -0.89373030
 H  1.43910775     0.00000000    -0.89373030
end

basis
 O library 6-31g*
 H library 6-31g*
end

occup 7 0  # single determinant
 1.75 
 0.25 
 2.0
 2.0
 2.0
 1.0
 1.0
end

dft
 print "final vectors analysis"
 xc xpbe96 cpbe96
end
task dft 
