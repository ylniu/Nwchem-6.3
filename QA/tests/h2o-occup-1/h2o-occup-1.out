 argument  1 = h2o-occup-1.nw



============================== echo of input deck ==============================

echo

start h2o-occup-1

memory 1000 mb

geometry units au noautosym noautoz
 O  0.00000000     0.00000000     0.23766060
 H -1.43910775     0.00000000    -0.89373030
 H  1.43910775     0.00000000    -0.89373030
end

basis
 O library 6-31g*
 H library 6-31g*
end

occup 7 0  # single determinant
 1.75
 0.25
 2.0
 2.0
 2.0
 1.0
 1.0
end

dft
 print "final vectors analysis"
 xc xpbe96 cpbe96
end
task dft
================================================================================


                                         
                                         


             Northwest Computational Chemistry Package (NWChem) 6.1.1
             --------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2010
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = orion
    program         = ../../../bin/LINUX64/nwchem
    date            = Fri Jun  1 12:23:34 2012

    compiled        = Fri_Jun_01_12:13:56_2012
    source          = /home/niri/nwchem/nwchem-trunk
    nwchem branch   = Development
    nwchem revision = 22009:22540M
    ga revision     = 9973
    input           = h2o-occup-1.nw
    prefix          = h2o-occup-1.
    data base       = ./h2o-occup-1.db
    status          = startup
    nproc           =        4
    time left       =     -1s



           Memory information
           ------------------

    heap     =   32768001 doubles =    250.0 Mbytes
    stack    =   32768001 doubles =    250.0 Mbytes
    global   =   65536000 doubles =    500.0 Mbytes (distinct from heap & stack)
    total    =  131072002 doubles =   1000.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------




                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in a.u. (scale by  1.000000000 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 O                    8.0000     0.00000000     0.00000000     0.22627818
    2 H                    1.0000    -1.43910775     0.00000000    -0.90511272
    3 H                    1.0000     1.43910775     0.00000000    -0.90511272

      Atomic Mass 
      ----------- 

      O                 15.994910
      H                  1.007825


 Effective nuclear repulsion energy (a.u.)       9.0877671457

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000


            XYZ format geometry
            -------------------
     3
 geometry
 O                     0.00000000     0.00000000     0.11974126
 H                    -0.76154308     0.00000000    -0.47896506
 H                     0.76154308     0.00000000    -0.47896506

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |       a.u.
 ------------------------------------------------------------------------------
    2 H                |   1 O                |     1.83059  |     1.83059
    3 H                |   1 O                |     1.83059  |     1.83059
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          2
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    2 H                |   1 O                |   3 H                |   103.65
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          1
 ==============================================================================



  library name resolved from: environment
  library file name is: </home/niri/nwchem/nwchem-trunk/src/basis/libraries/>
  
                      Basis "ao basis" -> "" (cartesian)
                      -----
  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  5.48467170E+03  0.001831
  1 S  8.25234950E+02  0.013950
  1 S  1.88046960E+02  0.068445
  1 S  5.29645000E+01  0.232714
  1 S  1.68975700E+01  0.470193
  1 S  5.79963530E+00  0.358521

  2 S  1.55396160E+01 -0.110778
  2 S  3.59993360E+00 -0.148026
  2 S  1.01376180E+00  1.130767

  3 P  1.55396160E+01  0.070874
  3 P  3.59993360E+00  0.339753
  3 P  1.01376180E+00  0.727159

  4 S  2.70005800E-01  1.000000

  5 P  2.70005800E-01  1.000000

  6 D  8.00000000E-01  1.000000

  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.87311370E+01  0.033495
  1 S  2.82539370E+00  0.234727
  1 S  6.40121700E-01  0.813757

  2 S  1.61277800E-01  1.000000



 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                           6-31g*                  6       15   3s2p1d
 H                           6-31g*                  2        2   2s


  pbe96 is a nonlocal functional; adding pw91lda local functional. 

                                 NWChem DFT Module
                                 -----------------


  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     3
          No. of electrons :    10
           Alpha electrons :     5
            Beta electrons :     5
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: off; symmetry adaption is: off
          Maximum number of iterations:  30
          AO basis - number of functions:    19
                     number of shells:    10
          Convergence on energy requested: 1.00D-06
          Convergence on density requested: 1.00D-05
          Convergence on gradient requested: 5.00D-04

              XC Information
              --------------
          PerdewBurkeErnzerhof Exchange Functional  1.000          
            Perdew 1991 LDA Correlation Functional  1.000 local    
           PerdewBurkeErnz. Correlation Functional  1.000 non-local

             Grid Information
             ----------------
          Grid used for XC integration:  medium    
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          O                   0.60       49           5.0       434
          H                   0.35       45           6.0       434
          Grid pruning is: on 
          Number of quadrature shells:   139
          Spatial weights used:  Erf1

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL): 1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho: 1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul: 1.00D-08


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:         -75.75081731

      Non-variational initial energy
      ------------------------------

 Total energy =     -75.898038
 1-e energy   =    -121.562760
 2-e energy   =      36.576956
 HOMO         =      -0.466838
 LUMO         =       0.111403

   Time after variat. SCF:      0.1
   Time prior to 1st pass:      0.1

 #quartets = 1.540D+03 #integrals = 8.874D+03 #direct =  0.0% #cached =100.0%


 Integral file          = ./h2o-occup-1.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   =  10690
 No. of bits per label  =      8        No. of bits per value  =     64


File balance: exchanges=     0  moved=     0  time=   0.0


 Grid_pts file          = ./h2o-occup-1.gridpts.0
 Record size in doubles =  12289        No. of grid_pts per rec  =   3070
 Max. records in memory =      7        Max. recs in file   =     57016


           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       32.55            32549332
          Stack Space remaining (MW):       32.77            32767784

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1    -76.2819047124 -8.54D+01  3.08D-01  4.28D-01     0.2
 d= 0,ls=0.0,diis     2    -69.0819144444  7.20D+00  5.83D-02  4.72D+00     0.2
 d= 0,ls=0.0,diis     3    -69.4428499034 -3.61D-01  5.21D-02  1.41D+00     0.3
 d= 0,ls=0.0,diis     4    -69.5814233998 -1.39D-01  2.05D-02  9.04D-02     0.3
 d= 0,ls=0.0,diis     5    -69.5911946806 -9.77D-03  1.74D-03  1.08D-03     0.3
 d= 0,ls=0.0,diis     6    -69.5913114705 -1.17D-04  3.93D-05  5.60D-07     0.4
 d= 0,ls=0.0,diis     7    -69.5913115327 -6.22D-08  1.12D-05  3.84D-08     0.4
 d= 0,ls=0.0,diis     8    -69.5913115368 -4.14D-09  2.26D-07  4.40D-11     0.5


         Total DFT energy =      -69.591311536828
      One electron energy =     -112.448053589101
           Coulomb energy =       42.191232924144
    Exchange-Corr. energy =       -8.422258017596
 Nuclear repulsion energy =        9.087767145726

 Numeric. integr. density =       10.000000766612

     Total iterative time =      0.4s



                       DFT Final Molecular Orbital Analysis
                       ------------------------------------

 Vector    1  Occ=1.750000D+00  E=-1.975227D+01
              MO Center= -1.5D-13, -1.2D-18,  1.2D-01, r^2= 1.5D-02
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.996065  1 O  s          

 Vector    2  Occ=2.500000D-01  E=-1.274324D+00
              MO Center= -1.1D-10, -4.7D-17, -9.8D-02, r^2= 4.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.545168  1 O  s                  6      0.430710  1 O  s          
     1     -0.215261  1 O  s                  5     -0.206675  1 O  pz         

 Vector    3  Occ=2.000000D+00  E=-7.613619D-01
              MO Center=  9.7D-11,  2.3D-17, -4.1D-02, r^2= 5.8D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.646270  1 O  px                 7      0.269163  1 O  px         
    16     -0.198895  2 H  s                 18      0.198895  3 H  s          

 Vector    4  Occ=2.000000D+00  E=-6.161499D-01
              MO Center= -2.8D-11,  3.9D-17,  2.7D-01, r^2= 5.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.663561  1 O  pz                 9      0.322213  1 O  pz         
     6      0.288313  1 O  s                  2      0.202220  1 O  s          

 Vector    5  Occ=2.000000D+00  E=-5.774555D-01
              MO Center= -1.4D-11,  2.9D-17,  9.3D-02, r^2= 4.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.747165  1 O  py                 8      0.386667  1 O  py         

 Vector    6  Occ=1.000000D+00  E=-9.083366D-02
              MO Center= -8.1D-10,  4.6D-17, -6.4D-01, r^2= 2.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.969067  1 O  s                 17     -0.827697  2 H  s          
    19     -0.827697  3 H  s                  9     -0.300719  1 O  pz         
     5     -0.295866  1 O  pz                 2      0.246462  1 O  s          
    16     -0.201648  2 H  s                 18     -0.201648  3 H  s          

 Vector    7  Occ=1.000000D+00  E=-1.024790D-02
              MO Center=  8.5D-10,  6.2D-18, -5.4D-01, r^2= 2.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17     -1.067848  2 H  s                 19      1.067848  3 H  s          
     7     -0.532094  1 O  px                 3     -0.470071  1 O  px         
    16     -0.239250  2 H  s                 18      0.239250  3 H  s          

 Vector    8  Occ=0.000000D+00  E= 5.615745D-01
              MO Center= -1.0D-08,  9.1D-17, -2.3D-01, r^2= 1.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17      0.850476  2 H  s                 19     -0.850476  3 H  s          
    16     -0.785782  2 H  s                 18      0.785782  3 H  s          
    12     -0.415209  1 O  dxz                7     -0.228369  1 O  px         
     3     -0.171534  1 O  px         

 Vector    9  Occ=0.000000D+00  E= 5.653908D-01
              MO Center=  9.9D-09,  1.4D-14,  1.7D-01, r^2= 1.5D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      0.826714  1 O  pz                 5     -0.797142  1 O  pz         
    17      0.546763  2 H  s                 19      0.546763  3 H  s          
    16     -0.448444  2 H  s                 18     -0.448444  3 H  s          
    10     -0.167955  1 O  dxx        

 Vector   10  Occ=0.000000D+00  E= 5.836347D-01
              MO Center= -8.5D-13, -1.4D-14,  1.1D-01, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      1.088566  1 O  py                 4     -0.881684  1 O  py         

 Vector   11  Occ=0.000000D+00  E= 6.705469D-01
              MO Center= -5.8D-11, -8.5D-17, -1.0D-01, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      1.188636  1 O  s                  2     -0.725823  1 O  s          
    16      0.675944  2 H  s                 18      0.675944  3 H  s          
     9      0.668352  1 O  pz                17     -0.554900  2 H  s          
    19     -0.554900  3 H  s                 13     -0.326900  1 O  dyy        
     5     -0.289444  1 O  pz                15     -0.196804  1 O  dzz        

 Vector   12  Occ=0.000000D+00  E= 7.788658D-01
              MO Center=  1.2D-10,  3.1D-16,  8.8D-02, r^2= 1.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      1.735963  1 O  px                17      1.037134  2 H  s          
    19     -1.037134  3 H  s                  3     -0.881703  1 O  px         

 Vector   13  Occ=0.000000D+00  E= 9.452625D-01
              MO Center=  1.2D-10, -2.8D-16, -3.4D-01, r^2= 1.5D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      3.551910  1 O  s                  2     -1.473143  1 O  s          
     9     -0.994630  1 O  pz                17     -0.807537  2 H  s          
    19     -0.807537  3 H  s                 10     -0.656093  1 O  dxx        
    15     -0.334509  1 O  dzz               16     -0.315355  2 H  s          
    18     -0.315355  3 H  s                  5      0.285339  1 O  pz         

 Vector   14  Occ=0.000000D+00  E= 1.385470D+00
              MO Center=  8.0D-12,  1.1D-15,  1.8D-01, r^2= 6.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      1.011549  1 O  dzz               10     -0.563877  1 O  dxx        
    13     -0.375239  1 O  dyy                6     -0.270014  1 O  s          

 Vector   15  Occ=0.000000D+00  E= 1.400484D+00
              MO Center=  2.1D-11, -6.4D-16,  1.2D-01, r^2= 6.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    11      1.732051  1 O  dxy        

 Vector   16  Occ=0.000000D+00  E= 1.426781D+00
              MO Center= -6.0D-12, -6.3D-16,  1.5D-01, r^2= 6.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    14      1.729626  1 O  dyz        

 Vector   17  Occ=0.000000D+00  E= 1.966526D+00
              MO Center=  4.7D-12, -3.8D-16, -2.5D-02, r^2= 8.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      1.660076  1 O  s                 13     -1.113200  1 O  dyy        
    16     -0.858526  2 H  s                 18     -0.858526  3 H  s          
     9     -0.793542  1 O  pz                10      0.728888  1 O  dxx        
     2     -0.559482  1 O  s                 17     -0.157032  2 H  s          
    19     -0.157032  3 H  s          

 Vector   18  Occ=0.000000D+00  E= 2.254515D+00
              MO Center=  2.5D-11,  4.0D-17,  5.4D-02, r^2= 9.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    12      2.187977  1 O  dxz               16     -0.955240  2 H  s          
    18      0.955240  3 H  s                  7     -0.879238  1 O  px         

 Vector   19  Occ=0.000000D+00  E= 3.143637D+00
              MO Center= -3.7D-12,  1.8D-17,  1.1D-01, r^2= 1.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      3.855584  1 O  s                 13     -1.600224  1 O  dyy        
    15     -1.571180  1 O  dzz               10     -1.548790  1 O  dxx        
    17     -0.591846  2 H  s                 19     -0.591846  3 H  s          
     1     -0.461846  1 O  s                  9     -0.362528  1 O  pz         
     2      0.194532  1 O  s          


 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.09965868

 moments of inertia (a.u.)
 ------------------
           2.291369192698           0.000000000000           0.000000000000
           0.000000000000           6.465843061885           0.000000000000
           0.000000000000           0.000000000000           4.174473869187

     Multipole analysis of the density
     ---------------------------------

     L   x y z        total         alpha         beta         nuclear
     -   - - -        -----         -----         ----         -------
     0   0 0 0      0.000000     -5.000000     -5.000000     10.000000

     1   1 0 0      0.000000      0.000000      0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000      0.000000
     1   0 0 1      0.682309      0.341154      0.341154      0.000000

     2   2 0 0     -9.650117     -6.896089     -6.896089      4.142062
     2   1 1 0      0.000000      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000      0.000000
     2   0 2 0     -6.236786     -3.118393     -3.118393      0.000000
     2   0 1 1      0.000000      0.000000      0.000000      0.000000
     2   0 0 2     -7.740795     -4.894434     -4.894434      2.048073


 Parallel integral file used       4 records with       0 large values


 Task  times  cpu:        0.5s     wall:        1.8s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  130      130     1715      291      803        0        0        0     
number of processes/call 1.45e+00 1.38e+00 1.28e+00 0.00e+00 0.00e+00
bytes total:             1.09e+06 2.48e+05 5.00e+05 0.00e+00 0.00e+00 0.00e+00
bytes remote:            3.29e+05 4.62e+04 1.62e+05 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 66440 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        23	        47
	current total bytes		         0	         0
	maximum total bytes		   1749216	  22510328
	maximum total K-bytes		      1750	     22511
	maximum total M-bytes		         2	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
  E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski, T. P. Straatsma,
          M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus, J. Hammond,
    J. Autschbach, F. Aquino, S. Hirata, M. T. Hackler, J. Mullin, P. Nichols,
    R. Peverati, Y. Zhao, P.-D. Fan, R. J. Harrison, M. Dupuis, D. M. A. Smith,
    J. Nieplocha, V. Tipparaju, M. Krishnan, A. Vazquez-Mayagoitia, L. Jensen,
      M. Swart, Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby,
        E. Brown, G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao,
        R. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell,
       D. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. Deegan,
        K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. Hess, J. Jaffe,
        B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield,
   X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing, K. Glaesemann,
          G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe,
                                A. Wong, Z. Zhang.

 Total times  cpu:        0.5s     wall:        3.0s
